cmd/cape/cmd/logout.go

```diff
@@ -0,0 +1,31 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+)
+
+var logoutCmd = &cobra.Command{
```

Comment:
This doesn't fully log someone out .. they should visit `login.capeprivacy.com/logout` to remove any browser state that they have.